src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,10 @@
-export const UserInfo = () => <>Put the user here</>;
+/* eslint-disable prettier/prettier */
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <>
+    <a key={user.id} className="UserInfo" href={`mailto:${user.email}`}>
```

Comment:
```suggestion
    <a className="UserInfo" href={`mailto:${user.email}`}>
```
we add key only to items in the map